sdk/bootstrap/main.go

```diff
@@ -251,6 +260,49 @@ func renderTemplates(templateMaxBytes int64) {
 	}
 }
 
+func installCerts() {
+	mesosSandbox := os.Getenv("MESOS_SANDBOX")
+	sslDir := filepath.Join(mesosSandbox, ".ssl");
+
+	sslDirExists, err := _pathExists(sslDir)
```

Comment:
I think you want to check if this is a dir, not if it exists.  In other words, right now this seems to be okay with a .ssl file, device file, named pipe, etc.